README.md

```diff
@@ -17,12 +17,14 @@ Create a new temporary project:
 
 * With no additional dependencies:
     `$ cargo-temp`
-
 * With multiple dependencies:
     `$ cargo-temp rand tokio`
-
-* With a dependency that have a fixed version:
-    `$ cargo-temp anyhow==1.0.13`
+* When specifying a version (You can use the 
+  [cargo's comparison requirements][comparison])
+    * `$ cargo-temp anyhow=1.0`
```

Comment:
This one is actually not a comparison requirement. See in the doc it is not under comparison requirements.